runtime/src/main/kotlin/kotlin/collections/HashMap.kt

```diff
@@ -7,6 +7,7 @@ package kotlin.collections
 
 import kotlin.native.concurrent.isFrozen
 
+@ShareByValue
```

Comment:
This class doesn't handle the case when a copied key's hash code is different from the original one's.